docs/overview/overview.md

```diff
@@ -1,45 +1,31 @@
 ---
 title: 'Office Scripts in Excel on the web'
-description: 'The prerequisites and environment information for Office Scripts for Excel on the web.'
-ms.date: 12/05/2019
+description: 'A brief introduction to the Action Recorder and Code Editor for Office Scripts.'
+ms.date: 12/12/2019
 localization_priority: Normal
 ---
 
-# Office Scripts in Excel on the web
+# Office Scripts in Excel on the web (preview)
 
 [!INCLUDE [Preview note](../includes/preview-note.md)]
 
-Office Scripts in Excel on the web let you automate tasks. You can record your Excel actions with the [Action Recorder](https://aka.ms/makersdogfood), which creates a script. You can also create and edit scripts with the Code Editor. This series of documents teaches you to use the Code Editor to make or update your own scripts by learning the fundamentals of Office Scripts.
+Office Scripts in Excel on the web let you automate tasks. You can record your Excel actions with the Action Recorder, which creates a script. You can also create and edit scripts with the Code Editor. This series of documents teaches you how to use these tools. You'll be introduced to the Action Recorder and see how to record your frequent Excel actions. You'll also learn how to make or update your own scripts with the Code Editor.
 
-> [!NOTE]
-> If you're looking to develop an Office Add-In for Excel, please visit our [quick start](/office/dev/add-ins/quickstarts/excel-quickstart-jquery) or [tutorial](/office/dev/add-ins/tutorials/excel-tutorial). Learn more about the differences between Office Scripts and Office Add-ins in [the Differences between Office Scripts and Office Add-ins article](../resources/add-ins-differences.md).
+!["A gif showing a script being recorded and replayed on different worksheets.](../images/teaser-animation.gif)
```

Comment:
Could the double quotes (") be causing a problem here?

Reply:
GIFs weren't being included by the build system. I'll put a PR out in a bit with a docfx config change.

Reply:
Ah, got it.